### ISSUE TITLE
Compare every thermal comfort sensor with outdoor thermal comfort

### ISSUE THREAD

**Issue by filikun** [dolezsa/thermal_comfort#140]
WHO has guidelines for indoor quality and states that there is a health risk if your indoor absolute humidity is greater than 3g/m3 of your outdoor absolute humidity. Therefore it would be an awesome feature to make it possible to compare your outdoor humidity with any indoor/room thermal comfort sensor.

You can read the report here https://apps.who.int/iris/handle/10665/164348
I found the info on the Swedish public health authority

**Comment by sevorl** [comment]
It is also useful to compare indoor vs outdoor humidity to decide wether you should open or close windows. 
If you want to avoid moist, open the windows when its dryer outside than inside. (comparing the abs. humidity)

**Comment by renarena** [comment]
Can we consider wind speed in sumammer to decide if it feels more comfortable to open or close the window if its warmer outside?

**Comment by Torgrima** [comment]
Good idea, as default it could select a weather sensor already active in home assistant. 

Outside temperature or temperature difference is also used for calculating energy need/energy loss. 

Degree day sensor: 
https://en.wikipedia.org/wiki/Degree_day

**Comment by renarena** [comment]
I would be interested in degree days too.
